### PR TITLE
My Country Info App

### PR DESCRIPTION
**Pull request by Mekuseo**
- The API should allow you to get numeric values based on a parameter
- Followed these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
Colors (select one primary color for your website).
Typography: font face, size, and weight.
Layout: composition and space between elements.
- When the page loads, the web app shows the list of categories that some parameter could filter out
- displayed some numeric values per category that come from the API.
- The application navigates to the details page when the user clicks (or taps) on a category item.
- In the details page, the web app retrieves data from the API to show detailed data in the category.
- When the user clicks on the "Back" button (<), the user navigates to the home page.
- Created unit tests for pure functions in your Redux code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,288 +1,175 @@
...
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
+### 🔥 Key Features
+### 🔥 Key Features
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
```
Comment (MahmoudBakr23) [inline README.md:114]: - [x] I know you might've forgotten, but it's ok. This only means you're 5 mins away from the approval 😉 Please record a 5 mins short presentation video to show off what you've done so far on this great capstone 💪

_Do not forget to attach the link proudly in the README file ❤️_

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Mekuseo 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪

- You nailed it in this great React app
- Well done structuring your files so good
- Great job using redux along with React hooks to handle the state of the app
- Well done working with API
- Kudos on the great filtering feature while fetching from the API plus the searching functionality
- Nicely designed
- Well-tested
- Descriptive PR and README
- Correct Linters
- Great job deploying the app

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Mekuseo) [reply README.md:114]: Hi @MahmoudBakr23 ,
Thank you for your reviews. However, the code extract you pointed out is an old code. It has been updated in the most recent PR.  The live demo link in the recent PR is in Line 56.
Please do kindly let me know If I am wrong in any way.
Thank you

Comment (MahmoudBakr23) [reply README.md:114]: Hi, @Mekuseo Thank you for communicating.

I can see the confusion here. However, I still don't see any link to the [video presentation](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=you%20should%20record%20a%20video%20presenting%20the%20features%20of%20the%20project%20you%20built.%20It%20is%20a%20video%20with%20a%20maximum%20length%20of%205%20minutes.) whether in the PR summary or the README file. On the other side of the confusion, I know that your app is deployed, accessed, and well-working and I highlighted this point in my review ⬆️ 😄

I hope this clears things out, please reach out again for further questions. 🙏

Comment (Mekuseo) [reply README.md:114]: Thank you for the clarifications. A link to the video presentation has been added.

File: src/services/CountriesService.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+const BASE_URL = 'https://restcountries.com/v3.1/';
```
Comment (amejid) [inline src/services/CountriesService.js:3]: - [ ] [OPTIONAL] You did great on the project, but the app is not working as expected. I believe the issue is that the API service providing the information is unavailable. I recommend you find another API that provides a better service. 🤝️

**Review by amejid — APPROVED** [review]
Hi @Mekuseo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Good test coverage :heavy_check_mark: 
- Great presentation :heavy_check_mark: 
- Professional README :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**